editor/editor_export_godot3.cpp

```diff
@@ -480,15 +464,12 @@ static const char *prop_renames[][2] = {
 };
 
 static const char *type_renames[][2] = {
-	{ "SpatialPlayer", "Spatial" },
-	{ "SpatialSamplePlayer", "Spatial" },
-	{ "SpatialStreamPlayer", "Spatial" },
-	{ "Particles", "Spatial" },
-	{ "SamplePlayer", "Node" },
-	{ "SamplePlayer2D", "Node2D" },
-	{ "SoundPlayer2D", "Node2D" },
-	{ "StreamPlayer2D", "Node2D" },
-	{ "Particles2D", "Node2D" },
+	{ "StreamPlayer", "AudioStreamPlayer" },
+	{ "SpatialSamplePlayer", "AudioStreamPlayer3D" },
+	{ "SpatialStreamPlayer", "AudioStreamPlayer3D" },
+	//{ "SamplePlayer", "Node" },
```

Comment:
So could you uncomment those two? Otherwise I think the nodes are discarded during the conversion (feel free to correct me if that's not the case :)).